lib/msf/ui/console/driver.rb

```diff
@@ -406,13 +406,25 @@ def on_startup(opts = {})
 
     run_single("banner") unless opts['DisableBanner']
 
+    av_warning_message if framework.eicar_corrupted?
+
     opts["Plugins"].each do |plug|
       run_single("load '#{plug}'")
     end if opts["Plugins"]
 
     self.on_command_proc = Proc.new { |command| framework.events.on_ui_command(command) }
   end
 
+  def av_warning_message
+      avdwarn = "\e[31m"\
+                "Warning: This copy of the Metasploit Framework has been corrupted by an installed anti-virus program."\
+                " We recommend that you disable your anti-virus or exclude your Metasploit installation path, "\
+                "then restore the removed files from quarantine or reinstall the framework.\e[0m"\
+                "\n\nFor more info: https://community.rapid7.com/docs/DOC-1273\n\n"
```

Comment:
This URL seems dead 👀

Reply:
So it is, uhhh any idea what it _used_ to point to?

Reply:
so it is, nice spot, don't suppose you know what it _used_ to say? :sweat_smile: 
I can either remove it or we can find/create a new page